beater/authorization/apikey.go

```diff
@@ -30,6 +30,9 @@ import (
 )
 
 const (
+	//DefaultResource for apm backend enabled API Keys
+	DefaultResource = "-"
```

Comment:
is this our convention or something special for elasticsearch?

Reply:
This is our convention. At the moment the API keys can be created with `resource: *` , but as soon as we support service specific API Keys, this resource needs to be enabled for the early key validation.